### PR TITLE
Ansible: Protection Job Path Update

### PR DESCRIPTION
**Pull request by naveena-maplelabs**
Issue:
> Existing module supports addition of new sources to already
available protection job.
> Updating include/exclude paths for an existing source id
is not available.

Changes:
> Check is made to find any changes in the include and exclude
paths, if detected source path is updated.

### CODE REVIEW

File: library/cohesity_job.py [hunk 1]
```diff
@@ -479,7 +480,7 @@ def register_job(module, self):
                 vms = module.params.get('exclude')
                 payload['excludeSourceIds'] = get_vmware_ids(module, parent_source_id, self, vms)
         data = json.dumps(payload)
-        # module.exit_json(output=data)
+        # module.exit_json(output=payload)
```
Comment (pyashish) [inline library/cohesity_job.py]: shouldn't this be json output instead of the object ?

File: library/cohesity_job.py [hunk 2]
```diff
@@ -532,7 +533,7 @@ def start_job(module, self):
         payload['runType'] = "k" + self['runType']
 
         data = json.dumps(payload)
-        # module.exit_json(output=data)
+        # module.exit_json(output=payload)
```
Comment (pyashish) [inline library/cohesity_job.py]: why are we changing the output type , can you give an example of how it was previously vs now ?

**Comment by pyashish** [comment]
@chandrashekar-cohesity  can you take a look at this review once.

**Comment by chandrashekar-cohesity** [comment]
@pyashish the new logic to handle updated path looks good to me. I tested it as well. I am not sure if I understand what you mean in the comments. 

@naveena-maplelabs can you answer/resolve the comments.

File: library/cohesity_job.py (hunk 1 above)
Comment (naveena-maplelabs) [reply library/cohesity_job.py]: @pyashish these are comments for debugging, payload contains payload for API request in json format.

**Pull request merged**